<?xml version="1.0"?>
<launch>
	 <node name="bhand_node" pkg="serial_bhand" type="bhand_node.py"/>
</launch>
